src/tools/perf/perftest.c

```diff
@@ -471,8 +471,9 @@ static ucs_status_t setup_sock_rte_p2p(struct perftest_context *ctx)
 
 err_close_connfd:
     ucs_close_fd(&connfd);
-    goto out_free_res;
 err_close_sockfd:
+    ucs_error("%s failed. %s",
```

Comment:
seems err_str could be un initialized when jumping to err_close_connfd

Reply:
it is initialized

Reply:
what about lines 425, 434, 443 for example?